### ISSUE TITLE
Setowner for normal player

### ISSUE THREAD

**Issue by BensaTV** [IntellectualSites/PlotSquared#3347]
### What feature do you want to see added?

Please add a permission that allows only changing the owner with /plot setowner on your own plot. Currently there is only an admin permission that allows a player to change the owner of anyones plot. This would take alot of work for me away as my wouldn't have to ask me every single time if I can change the owner of their plot. This also used to be a thing in PlotSquared V3 and even V4 I think.

### Are there any alternatives?

Not really. I could use an outdated version of plotsquared but I don't really see that as an option as I paid for this one and want to avoid Bugs.

**Comment by zOIDAdaJulian** [comment]
I would also apreciate this feature very much! 😄
Definitely a upvote from my side 👍

**Comment by NotMyFault** [comment]
I'm a bit opposed to the suggested implementation. `Setowner` is intended to be an administrative command, it does not respect the same limits other commands do, and does not fire events you would expect it to do.
An in my opinion better approach would be something like `/plot transfer`, that transfers the plot ownership from one user to another, while respecting economy, grants and possible third party circumstances (e.g. firing separate events; at this time it's not possible to distinguish whether a `setowner` action has been instantiated as administrative or user action), everything setowner does not do.
An optional confirmation, like for plot merged with different owners, should be added as well, considering it's an irreversible action for the actor.
The existing restriction logic from `setowner` can be removed in the same run.

**Comment by zOIDAdaJulian** [comment]
Yeah of course 👍 Thanks for looking into the request, I really think a command like a transfer command would be a huge innovation for PlotSquared 😁

**Comment by DueKay** [comment]
Evening together,

I wanted to ask if there will ever be the possibility for users to hand in their own plots? Whether with /p give or /p transfer
But it would make everything much more pleasant than if you always have to transfer it to someone for the users if they want it.
As already mentioned, the limitation of the plots should be taken into account so that nobody can claim more than they are allowed to. And I also find the possibility that the other user has to accept the transmission of the plot very useful.

Thanks in advance and best regards

Ps. Sry for the probably bad grammar. I used google translator xD


------------------------------------------------------------------------------------------------------


Abend zusammen,

Ich wollte mal nachfragen, ob es irgendwann die Möglichkeit für User geben wird, dass sie selbstständig ihr Plot abgeben können? Egal ob mit /p give oder /p transfer
Aber es würde alles doch wesentlich angenehmer machen, als wenn man es immer für die User an jemanden übertragen muss, wenn diese das gerne möchten.
Wie schon erwähnt sollte die begrenzung der Plots hierfür berücksichtigt sein, dass niemand dadurch mehr in anspruch nehmen kann, als er darf. Und auch die Möglichkeit das der andere User die Übertragung des Plots akzeptieren muss, finde ich sehr sinnvoll.

Danke schonmal und liebe Grüße

Ps. Sry für die wahrscheinlich schlechte Grammatik. Ich hab Google Übersetzer benutzt xD

**Comment by BensaTV** [comment]
We really need this.

**Comment by BensaTV** [comment]
It's been almost two years. Is there any update to this?

**Comment by NeoFoxi** [comment]
please add this and multiple owners this would be awesome!

**Comment by benjissuperaccount** [comment]
Multiple owners exist through merging plots of different players so this should be definitely possible. Am 18.08.2026 um 18:37 schrieb NeoFoxi ***@***.***>:﻿NeoFoxi left a comment (IntellectualSites/PlotSquared#3347)
please add this and multiple owners this would be awesome!

**Comment by NeoFoxi** [comment]
> Multiple owners exist through merging plots of different players so this should be definitely possible. Am 18.08.2026 um 18:37 schrieb NeoFoxi ***@***.***>:﻿NeoFoxi left a comment ([IntellectualSites/PlotSquared#3347](https://github.com/IntellectualSites/PlotSquared/issues/3347))
> please add this and multiple owners this would be awesome!
> 
> —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

but it's marked as a bug and I dont want it bicouse it could break in the future if it gets fixed or so when a bug gets fixed no one cares about the users that used it but when a feature gets changed migrators are built and compatibility for the olds and so one.
